draft-ietf-homenet-front-end-naming-delegation.txt

```diff
@@ -1111,17 +1135,6 @@ Internet-Draft                public-names                     July 2021
    This section details security policies related to the Hidden Primary
    / Secondary synchronization.
 
-
-
-
-
-
-
-Migault, et al.          Expires 14 January 2022               [Page 20]
-
-Internet-Draft                public-names                     July 2021
-
-
    The HNA, as Hidden Primary SHOULD drop any DNS queries from the home
    network.  This could be implemented via port binding and/or firewall
    rules.  The precise mechanism deployed is out of scope of this
```

Comment:
1148: remove "At the DNS level,"
1160: you say the document "assumes" HNA signs. wouldn't "recommended" be a better term here? i think having an explicit architecture/design decision is better than assumption.

Reply:
changed to It is RECOMMENDED